crates/cubecl-std/src/tensor/virtual.rs

```diff
@@ -407,5 +424,78 @@ mod __tensor {
         fn __expand_buffer_len_method(&self, scope: &mut Scope) -> ExpandElementTyped<u32> {
             self.clone().__expand_buffer_len_method(scope)
         }
+
+        fn __expand_as_tensor_map_method(
+            &self,
+            _scope: &mut Scope,
+        ) -> ExpandElementTyped<TensorMap<E>> {
+            todo!()
+        }
+    }
+}
+
+/// Enable tensor maps to be virtual.
+mod __tensor_map {
```

Comment:
I would remove that if not necessary.

Reply:
It's necessary because all the matmul stuff needs virtual tensors. So TensorMaps need to be virtual too.